presets/4.5/tune/aos_rc/aos_65mm_tune.txt

```diff
@@ -0,0 +1,129 @@
+#$ TITLE: 65mm Tiny whoop tune by Chris Rosser
+#$ FIRMWARE_VERSION: 4.5
+#$ CATEGORY: TUNE
+#$ STATUS: COMMUNITY
+#$ KEYWORDS: aos, chris, rosser, pid, tune, 65mm, tiny, whoop
+#$ AUTHOR: Chris Rosser
+#$ DISCUSSION: https://www.aos-rc.com
+#$ DESCRIPTION: Custom Tune for a 65mm Tiny whoop. Based on a 300mAh 1S build with 23000KV Motors.
+#$ DESCRIPTION:
+#$ DESCRIPTION: Master multiplier is set to 1.3 for safety. Increase it carefully to further improve flight performance.
+#$ DESCRIPTION:
+#$ DESCRIPTION: RPM filtering is MANDATORY!
+#$ DESCRIPTION:
+#$ DESCRIPTION: Please carefully review the options above and tick any that are relevant.
+#$ DESCRIPTION:
+#$ DESCRIPTION: Choose your RC link speed.
+#$ DESCRIPTION:
+#$ FORCE_OPTIONS_REVIEW: TRUE
+
+#$ INCLUDE: presets/4.5/tune/defaults.txt
+
+# -- PID values --
+set profile_name = AOS 65mm
+set p_pitch = 95
+set i_pitch = 170
+set d_pitch = 44
+set f_pitch = 194
+set p_roll = 76
+set i_roll = 135
+set d_roll = 39
+set f_roll = 156
+set p_yaw = 76
+set i_yaw = 135
+set f_yaw = 156
+set d_min_roll = 39
+set d_min_pitch = 44
+set angle_p_gain = 100
```

Comment:
since you using sliders for PIDs, you don't have to list all PID values.
(and because you used `simplified_tuning apply`)

Reply:
changed 👍